docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -0,0 +1,122 @@
+# Configuring nRF Connect SDK examples
+
+The nRF Connect SDK example applications all come with a default configuration for building.
+Check the information on this page if you want to modify the application configuration or
+add new functionalities to build your own application based on the provided example.
+This page also contains information about the configuration structure, which can be useful
+to better understand the building process.
+
+This guide can be used with the following examples:
+
+- [CHIP nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md)
+- [CHIP nRF Connect Lighting Example Application](../../examples/lighting-app/nrfconnect/README.md)
+- [CHIP nRF Connect Pigweed Example Application](../../examples/pigweed-app/nrfconnect/README.md)
+
+<hr>
+
+## Configuring application
+
+Changing the default application configuration can be done either temporarily or permanently.
+Changing configuration temporarily is useful for testing the impact of changes
+on the application behavior. Making permanent changes is better if you want
+to develop your own application, as it helps avoid repeating the configuration process.
+
+<hr>
+
+### Temporary changes to configuration
+
+You can change the configuration temporarily by editing the `*.config` file in the `build/zephyr/` directory,
+which stores all configuration options for the application generated as a result of the build process.
+As long as you do not remove the current build directory or delete this file, your changes will be kept.
+However, if you do a clean build, your changes are gone, so it is not possible to save changes permanently this way.
+
+Complete the following steps:
+1. Build the application by typing the following command in the example directory, with *board_name*
+   replaced with the target board name, for example *nrf52840dk_nrf52840*:
+
+        $ west build -b board_name
+
+2. Run the terminal-based interface called menuconfig by typing the following command:
+
+        $ west build -t menuconfig
+
+   The menuconfig terminal window appears, in which you can navigate using arrow keys and other keys,
+   based on the description at the bottom of the window.
+3. Make the desired changes by following the menuconfig terminal instructions.
+4. Press `Q` to save and quit.
+5. Rebuild the application.
+
+At this point, the configuration changes are applied to the output file and it can be
+flashed to the device.
+
+<hr>
+
+### Permanent changes to configuration
+
+The permanent solution is based on modifying the Kconfig files, which are used as
+components of the building process. This makes the changes persistent across builds.
+
+The best practice to make permanent changes is to edit the main application configuration file
+`prj.conf`, which is located in the example directory.
+This will result in overriding the existing configuration values.
```

Comment:
I'm not quite sure what you mean. I don't remember if I was writing this sentence or it appeared in the result of review, but for me the first one paragraph is about generating build output using configuration files and the second one is about modifying these files. So the first sentence informs that modifying file is permanent way of changing configuration and is not disappearing after clean and the second informs that modifying prj.conf file allows to override values assigned in the other files.

Reply:
Since up to this point there was nothing about the configuration structure I assumed that the sentence means that you're going to override the existing project's configuration and I thought it's the same as saying that you "make persistent changes across builds".